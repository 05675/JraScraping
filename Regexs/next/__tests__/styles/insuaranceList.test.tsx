import InsuaranceList from '../../../payroll/src/styles/insuaranceList';
import React from 'react';
import renderer from 'react-test-renderer';

describe('InsuaranceList', () => {
  test('InsuaranceList_snapshot', () => {
    const props = { listItems: [{ kind: '保険種別', firm: '保険会社名', key: 0 }] };
    const content = renderer.create(<InsuaranceList {...props} />).toJSON();
    expect(content).toMatchSnapshot();
  });
});
