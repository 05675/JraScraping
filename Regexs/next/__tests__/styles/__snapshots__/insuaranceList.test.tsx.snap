// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsuaranceList InsuaranceList_snapshot 1`] = `
<div
  className="sc-AxjAm eqKZAZ"
>
  <div
    className="sc-AxirZ hLpRhP"
  >
    <div>
      保険種別
    </div>
    <div
      className="sc-AxiKw jESKZa"
    >
      保険会社名
    </div>
  </div>
  <div
    className="sc-AxirZ sc-AxhCb tTVXF"
  >
    →
  </div>
</div>
`;
